style.css

```diff
@@ -0,0 +1,249 @@
+html {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  width: 100%;
+}
+
+*,
+*:before,
+*:after {
+  background-repeat: no-repeat;
+  -webkit-box-sizing: inherit;
+          box-sizing: inherit;
+}
+
+a {
+  text-decoration: none;
+}
+
+body {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+nav {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+  padding: 0 1rem;
+  background-color: black;
+  color: white;
+}
+
+nav p {
+  font-size: 1.7rem;
+  margin: 0.8rem 0;
+}
+
+nav ul {
+  list-style-type: none;
+}
+
+nav ul li {
```

Comment:
Why not try applying BEM naming strategy in order to make your naming conventions easier to interprete.